src/configure/helper/LocalGitRepoHelper.ts

```diff
@@ -17,10 +17,16 @@ export class LocalGitRepoHelper {
     private constructor() {
     }
 
-    public static GetHelperInstance(repositoryPath: string): LocalGitRepoHelper {
-        var repoService = new LocalGitRepoHelper();
-        repoService.initialize(repositoryPath);
-        return repoService;
+    public static async GetHelperInstance(repositoryPath: string): Promise<LocalGitRepoHelper> {
+        try {
+            var repoService = new LocalGitRepoHelper();
+            repoService.initialize(repositoryPath);
+            await repoService.gitReference.status();
+            return repoService;
```

Comment:
Can you check if we are handling repos without a remote and with a remote that is not supported

Reply:
yes we are doing that